packages/next/pages/_document.tsx

```diff
@@ -440,6 +447,7 @@ export class NextScript extends Component<OriginProps> {
           }${_devOnlyInvalidateCacheQueryString}`}
           nonce={this.props.nonce}
           crossOrigin={this.props.crossOrigin || process.crossOrigin}
+          {...modernProps}
```

Comment:
I think you need to create two script tags here (type="module", nomodule).

Reply:
Hey azizhk. The scripts and dynamic chunks for a page are read from the build manifest. So both module/nomodule scripts will be iterated over and added